go-controller/pkg/node/node.go

```diff
@@ -468,6 +473,19 @@ func (n *OvnNode) Start(wg *sync.WaitGroup) error {
 		}
 	}
 
+	// remove the NoSchedule Taint from the node, now that networking setup is done.
+	if nodeObjJson, err := json.Marshal(node); err == nil {
```

Comment:
@tssurya I think I'd like to see these (add taint, remove taint) as separate functions with the `if x, err := Foo(); err != nil { return }` pattern. Makes the already long code in Start() more readable.

Reply:
done.